x/stakeibc/keeper/msg_server.go

```diff
@@ -1103,3 +1103,24 @@ func (k msgServer) ResumeHostZone(goCtx context.Context, msg *types.MsgResumeHos
 
 	return &types.MsgResumeHostZoneResponse{}, nil
 }
+
+// Registers a community pool rebate, configuring the rebate percentage and liquid stake amount
+func (k msgServer) RegisterCommunityPoolRebate(
```

Comment:
So this function is a strict overwrite... avoids the extra crud functions or needing to make an add to stake or similar.  If the comm pool does another stake in the future we would just run this again with the updated current total amount.

Reply:
yeah that was my thinking